File: Python/abstract-factory/sample02/PcFactory.py
from BaseFactory import BaseFactory
from PcCPU import PcCPU
from PcRAM import PcRAM

class PcFactory(BaseFactory):
    @staticmethod
    def produceCPU():
        return PcCPU()

    @staticmethod
    def produceRAM():
        return PcRAM()

File: Homework/AbstractFactory/ch6/PcFactory.java
package ch6;

/**
 * Created by RayYu on 2016/12/7.
 */
public class PcFactory implements AbstractFactory {
    public CPU getCPU(){
        return new PcCPU();
    }
    public RAM getRAM(){
        return new PcRAM();
    }
}
